plugins/emc/emc_vnx_block_lun_perfdata

```diff
@@ -0,0 +1,391 @@
+#!/bin/bash
+
+: <<=cut
+
+=head1 NAME 
+
+ emc_vnx_block_lun_perfdata - Plugin to monitor Block statistics of EMC VNX 5300 Unified Storage Processors
+
+=head1 AUTHOR
+
+ Evgeny Beysembaev <megabotva@gmail.com>
+
+=head1 LICENSE
+
+ GPLv2
+
+=head1 MAGIC MARKERS
+
+  #%# family=auto
+  #%# capabilities=autoconf
+
+=head1 DESCRIPTION
+
+ The plugin monitors LUN of EMC Unified Storage FLARE SP's. Probably it can also be compatible with 
+ other Clariion systems. It uses SSH to connect to Control Stations, then remotely executes 
+ /nas/sbin/navicli and fetches and parses data from it. Obviously, it's easy to reconfigure plugin not to use 
+ Control Stations' navicli in favor of using locally installed /opt/Navisphere's cli. There is no difference which
+ Storage Processor to use to gather data, so this plugin tries both of them and uses the first active one.
+ This plugin also automatically chooses Primary Control Station from the list by calling /nasmcd/sbin/getreason and 
+ /nasmcd/sbin/t2slot.
+ 
+ I left some parts of this plugin as rudimental to make easy to reconfigure it to draw more (or less) data.
+
+=head1 COMPATIBILITY
+
+ The plugin has been written for being compatible with EMC VNX5300 Storage system, as this is the only EMC storage which 
+ i have. By the way, i am pretty sure it can also work with other VNX1 storages, like VNX5100 and VNX5500, and old-style 
+ Clariion systems.
+ About VNX2 series, i don't know whether the plugin will be able to work with them. Maybe it would need some corrections
+ in command-line backend. The same situation is with other EMC systems, so i encourage you to try and fix the plugin. 
+ 
+=head1 CONFIGURATION
+
+=head2 Prerequisites
+
+ First of all, be sure that statistics collection is turned on. You can do this by typing:
+ navicli -h spa setstats -on
+ on your Control Station or locally through /opt/Navisphere 
+
+ Also, the plugin actively uses buggy "cdef" feature of Munin, and here we can be hit by the following bugs:
```

Comment:
Please do not copy the workaround ("change value in file ...") from the ticket here - just use a link. Otherwise this information could get outdated, when this problem can be treated differently.
Please also detail what exactly could break due to this issue (My guess: only the "load" graphs would not work. Also see my comment further below regarding the usage of "cdef" in some other situations). Otherwise potential users may get discouraged to use this plugin at all, if they do not want to (or cannot) patch their munin installation.